build.gradle

```diff
@@ -89,7 +89,7 @@ dependencies {
 
     annotationProcessor "org.littletonrobotics.akit.junction:junction-autolog:3.0.2"
 
-    implementation 'com.github.galaxia5987:common:906c24dfca'
+    implementation 'com.github.galaxia5987:common:b0750b36da'
```

Comment:
Release a new release instead.

Reply:
@vichik123 said to wait with creating a new release

Reply:
What's the reason?

Reply:
I just don't want a million releases. Once the code is more compiled we'll take out a new release.